PBM-1213 Doc-restore-users-roles (#171)
percona/pbm-docs: docs/features/selective-backup.md

During the selective restore, the primary shard for a database is set to the state it had during the backup. For example, the primary shard for the database "Staff" during backup was A. After you restore the  "Staff" database, the primary shard will be set to A even if you moved the primary from A to B before the restore. All non-sharded collections will be restored on A; however, they will not be deleted from B. You must take needed actions (cleanup or move the primary back to B) to maintain them. 

## Restore a database with users and roles

!!! admonition "Version added: [2.5.0](../release-notes/2.5.0.md)"

You can restore the specified databases with users and roles that were created against them. This feature is useful for deployments where each user has an individual database and authenticates against it. In such a way, you can recover desired datasets to their state prior to data corruption or loss.

Consider these specifics of selective restore with users and roles:

> * Users and roles must be created in custom databases. For security considerations, users created in `admin` database are not restored.

Reviewer: ```suggestion
* Users and roles must be created in custom databases. For security considerations, users created in `admin`, `config` or `local` cannot be part of selective restore.
```

Btw, this restriction comes from selective restore and it is just inherit in users&roles use case. That means, that it is not possible to restore those 3 databases even without users&roles.
